src/pageObjects/components/SignUpForm.js

```diff
@@ -0,0 +1,74 @@
+import BaseComponent from "../BaseComponent.js";
+import {expect} from "@playwright/test";
+
+export default class SignUpForm extends BaseComponent {
+
+    _inputName = '#signupName';
+    _inputLastName = '#signupLastName';
+    _inputEmail = '#signupEmail';
+    _inputPassword = '#signupPassword';
+    _inputRepeatPassword = '#signupRepeatPassword';
+    _registerButton = '.modal-footer button';
+    _alertDanger = '.alert-danger';
+    _errors = '.invalid-feedback';
+
+    constructor(page, inputValues) {
+        super(page, page.locator('div.modal-content'));
+        this.inputValues = inputValues;
+        this.inputName = this._container.locator(this._inputName);
+        this.inputLastName = this._container.locator(this._inputLastName);
+        this.inputEmail = this._container.locator(this._inputEmail);
+        this.inputPassword = this._container.locator(this._inputPassword);
+        this.inputRepeatPassword = this._container.locator(this._inputRepeatPassword);
+        this.registerButton = this._container.locator(this._registerButton);
+        this.alertDanger = this._container.locator(this._alertDanger, {hasText: 'User already exists'});
+        this.errors = this._container.locator(this._errors);
+    }
+
+    #inputs = () => {
+        return {
+            signupName: this.inputName,
+            signupLastName: this.inputLastName,
+            signupEmail: this.inputEmail,
+            signupPassword: this.inputPassword,
+            signupRepeatPassword: this.inputRepeatPassword
+        }
+    }
```

Comment:
why?

Reply:
okay, I see. It's not bad but overcomplicated IMO.

Reply:
I can replace it